packages/solana/src/utils/wormhole/coder/idl.ts

```diff
@@ -54,12 +54,6 @@ export class IdlCoder {
       case 'i128': {
         return borsh.i128(fieldName);
       }
-      case 'u256': {
```

Comment:
Why are we removing these? Are they not used anywhere?

Reply:
I don't think they're used and they puke when I try to build docs.

See https://github.com/nonergodic/sdkv2/pull/3#discussion_r1268194985